### ISSUE TITLE
cellxgene_census_builder error when use manifest parameter

### ISSUE THREAD

**Issue by Liripo** [chanzuckerberg/cellxgene-census#498]
I want to try to build cellxgene-census with my own dataset, follow the  [README](https://github.com/chanzuckerberg/cellxgene-census/tree/main/tools/cellxgene_census_builder#readme), 
![image](https://github.com/chanzuckerberg/cellxgene-census/assets/54770415/623b987b-8ad6-4522-9c99-5e22110a3a7f)
but get an error.

**my code**: 
```bash
python -m cellxgene_census_builder ./ build --manifest ./manifest.csv
```
**error**:
```bash
usage: cellxgene_census_builder [-h] [--test-resume | --no-test-resume] working_dir
cellxgene_census_builder: error: unrecognized arguments: build --manifest ./manifest.csv
```

**Comment by Liripo** [comment]
Yes, I've had a rough look at [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA/tree/main)，But it seems to only provide query of a single data set, and for building own database, there should be other operational?

My original purpose was just to learn how to build a database and quickly query the expression of genes in the database. Of course, if I master it, I may apply it to some related projects.

**Comment by brianraymor** [comment]
Closed as _answered_.
